packages/block-library/src/calendar/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Server-side rendering of the `core/calendar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/calendar` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content.
+ */
+function render_block_core_calendar( $attributes ) {
+	global $monthnum, $year;
+	if ( isset( $attributes['monthNum'] ) ) {
+		$monthnum = $attributes['monthNum'];
+	}
+
+	if ( isset( $attributes['year'] ) ) {
+		$year = $attributes['year'];
+	}
+	$custom_class_name = empty( $attributes['className'] ) ? '' : ' ' . $attributes['className'];
+	$align_class_name  = empty( $attributes['align'] ) ? '' : " align{$attributes['align']}";
+
+	return sprintf(
+		'<div class="%1$s">%2$s</div>',
+		esc_attr( 'wp-block-calendar' . $custom_class_name . $align_class_name ),
+		get_calendar( true, false )
+	);
+}
+
+/**
+ * Registers the `core/calendar` block on server.
+ */
+function register_block_core_calendar() {
+	register_block_type(
+		'core/calendar',
+		array(
+			'attributes'      => array(
+				'align'     => array(
+					'type' => 'string',
+				),
+				'className' => array(
+					'type' => 'string',
+				),
+				'monthNum'  => array(
```

Comment:
We generally try to avoid abbreviations like 'num'. Could we get away with simply naming this `month`?